pkgs/development/libraries/polkit/default.nix

```diff
@@ -25,6 +25,7 @@
 , elogind
 # needed until gobject-introspection does cross-compile (https://github.com/NixOS/nixpkgs/pull/88222)
 , withIntrospection ? (stdenv.buildPlatform == stdenv.hostPlatform)
+, withGtkDoc ? (stdenv.buildPlatform == stdenv.hostPlatform)
```

Comment:
nit: might be worth adding a short comment about what fails (polkit-1-scan) for future reference.

Reply:
@pennae done